src/containers/FormikForm/formikDraftHooks.js

```diff
@@ -32,35 +33,63 @@ export function useFetchArticleData(articleId, locale) {
     if (articleId) {
       setLoading(true);
       const article = await draftApi.fetchDraft(articleId, locale);
+
+      const convertedConcepts = await fetchElementList(article.conceptIds);
       const taxonomy = await fetchTaxonomy(articleId, locale);
       setArticle(
-        transformArticleFromApiVersion({ taxonomy, ...article }, locale),
+        transformArticleFromApiVersion(
+          { taxonomy, ...article },
+          locale,
+          convertedConcepts,
+        ),
       );
       setLoading(false);
     }
   };
 
   const updateArticle = async updatedArticle => {
-    const savedArticle = await draftApi.updateDraft(updatedArticle);
+    const conceptIds = updatedArticle.conceptIds.map(concept => concept.id);
+    const savedArticle = await draftApi.updateDraft({
+      ...updatedArticle,
+      conceptIds: conceptIds,
+    });
     const taxonomy = await fetchTaxonomy(articleId, locale);
     const updated = transformArticleFromApiVersion(
       { taxonomy, ...savedArticle },
       locale,
+      updatedArticle.conceptIds,
     );
+    console.log(updated);
```

Comment:
Fjerne console.log.

Reply:
Kunne no-console vært satt til 'warn' i eslint? Merker jeg litt for ofte pusher console.log.

Reply:
Det kan du godt teste ut.